docs/ADMIN_BRIEF.md

```
@@ -0,0 +1,1218 @@
+  - Add comments/feedback on submissions
+  - Approve or reject submissions (but cannot directly publish)
+  - View blog posts and comments (read-only)
+  - Cannot edit website settings or user data
+  - Cannot delete content
+  - Cannot manage categories
+
+### 1.3 Permission Matrix
+
+| Feature | Super Admin | Editor | Moderator | Notes |
+|---------|-------------|--------|-----------|-------|
+| **Tool Management** |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 5; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)